recipe/meta.yaml

```
@@ -40,11 +42,11 @@ requirements:
     - llvmlite 0.40.0
     - numpy {{ numpy_min_ver }}
     - tbb-devel 2021.8.0
-    - _openmp_mutex            # [linux]
-    - importlib_metadata       # [py<39]
+    - _openmp_mutex 5.1        # [linux]
+    - importlib_metadata 6.0.0 # [py<39]
```

```suggestion
    - importlib_metadata  # [py<39]
```
For python packages in host, we can drop pinnings

---

Does it not matter that this will trigger a linter error?

---

I also don't get it. In the recipe the version is not specified, but this is often the case. @skupr-anaconda why shall we remove this one?